kiosk/src/Kiosk.css

```diff
@@ -327,28 +346,39 @@ h2 {
 /* ----------------------------------- */
 
 .carouselWrap {
-    width: 100%;
+    width: 99%;
     height: 80vh;
-    margin-top: 2rem;
-    display: flex;
-    flex-direction: column;
-    align-items: baseline;
-    /* justify-content: center; */
-    margin-right: -2rem
+    border-radius: 1.5rem;
+}
+
+.selected {
```

Comment:
```suggestion
.swiper-slide-active .selected {
```

would this work to make it so just the center one has the 'selected' border? when all the tiles have the same border it transitions a bit in my head from a selected indicator to just a normal border someone decided to set, but that could just be me~

Reply:
Oh, yeah, this does work. I was having problems with this when I was trying to have the border around the whole carousel, but when having the border around the game tile this works well. 

My worry is that there isn't enough visual distinction to really tell the user that the game is selected over the button, but I could be wrong.

You can play with it here: https://9c00-99-178-117-12.ngrok.io/ (my local host serve, ported by ngrok)
Having the one game selected is very obvious with games with dark thumbnails, but a bit more discreet with the more colorful games.

![selected-carousel3](https://user-images.githubusercontent.com/49178322/205363779-7994d5ce-ff7b-4f1b-9efa-4e7feb5bd9b7.gif)

Reply:
This is similar to how it is if all the games have 'bright' background right (they'd all have the border but it's not super distinct).

I wonder if a 1 or 2px black border would be enough to help out their without breaking the aesthetic~
<img width="1302" alt="image" src="https://user-images.githubusercontent.com/5615930/205365048-9204361f-e30f-4bca-ad59-5159f47a496f.png">
 or 2px
<img width="1299" alt="image" src="https://user-images.githubusercontent.com/5615930/205364945-3fa841f1-0a8c-4597-9391-ef9b42a4321a.png">